package.json

```diff
@@ -72,5 +70,8 @@
       }
     ]
   },
-  "dependencies": {}
+  "dependencies": {},
+  "engines": {
+    "node": ">=14.5.0"
```

Comment:
Great addition but does it need to be so high? Can we get away with v10 or 12 instead?

Reply:
We would need at least v13 where esmodules were no longer behind a flag
but this is only required by use the developers. ultimately you run this against a build so it actually isn't really a problem I guess it can be removed.

Should I remove it?

Reply:
I think we can remove, there is still a builder in place and I don't want to force ESM on others.